lib/signal_reader/parsers/iso15693/iso15693_parser.c

```diff
@@ -26,6 +26,7 @@ typedef enum {
 struct Iso15693Parser {
     Iso15693ParserState state;
     Iso15693ParserMode mode;
+    bool detect_mode;
```

Comment:
Can this detect_mode be moved to the Iso15693ParserMode enum?

Reply:
Sure, so you're proposing to add two new modes like `Iso15693ParserModeForced1OutOf4` and `Iso15693ParserModeForced1OutOf256` in place of the boolean?

Reply:
Yes, but there is a more important thing, which I've missed previously. Putting protocol specific functions into nfc.c / nfc.h layer it's not very good idea, because nfc layer was designed to be a generic protocol free. I had a quick conversation with original nfc stack designer and he agreed on that.
Also, we came up to the idea that we need to investigate this problem deeper and try to fix this in iso15 parser. Currently I'm investigating this issue and will be in touch with you on the results.